ui_framework/components/tool_bar/tool_bar_search_box.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, mount } from 'enzyme';
+import sinon from 'sinon';
+import { commonHtmlProps } from '../../test/common_html_props';
+
+import {
+  KuiToolBarSearchBox,
+} from './tool_bar_search_box';
+
+describe('KuiToolBarSearchBox', () => {
```

Comment:
I think that consistency will be difficult in larger components, as you might not be able to test each prop separately. E.g. when you `if` based on a prop and inside that do something based on another prop. Then it doesn't make sense separating like that, so the tests then have to be focused at the domain logic (aka the tasks the component solves) instead. As long as it's just these super-simple HTML element wrappers doing it this way works, though.

I'm in favor of removing the `describe('Props',` level, at least.